packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -73,7 +73,7 @@ export class ChainService implements ChainServiceInterface {
   private readonly blockConfirmations: number;
   private transactionQueue = new PQueue({concurrency: 1});
 
-  private finalizingChannels: {finalizesAtS: number; channelId: Bytes32}[] = [];
+  protected finalizingChannels: {finalizesAtS: number; channelId: Bytes32}[] = [];
```

Comment:
See my comment around `evm_mine` on how we can avoid upgrading `finalizingChannels` to `protected`